packages/datadog-plugin-kafkajs/src/index.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+function createWrapProducer (tracer, config) {
+  return function wrapProducer (createProducer) {
+    return function producerWithTrace () {
+      const serviceName = config.service || `${tracer._service}-kafka`
+      const createdProducer = createProducer.apply(this, arguments)
+
+      const send = createdProducer.send
+
+      const tags = {
+        'service.name': serviceName,
+        'span.kind': 'producer',
+        'span.type': 'queue',
```

Comment:
For the producer, this should be different than the consumer. Since the producer acts as a client to the broker, this could be `client`.